resources/jquery.ui/jquery.ui.slider.js

```diff
@@ -167,8 +167,13 @@ $.widget( "ui.slider", $.ui.mouse, {
 			return false;
 		}
 
+		var outerWidth = this.element.outerWidth();
+			//IE11 iframe fullscreen fix (KMS-4606)
+		if( !mw.getConfig('EmbedPlayer.IsFriendlyIframe') && window["resizeScrubberIE11"] === true ){
```

Comment:
@GuyLivni i dont like this fix - once we want to update jquery.ui.scrubber we'll forget it...
please consult @amirch1 on it